src/rules/createKeyframeRule.js

```diff
@@ -0,0 +1,44 @@
+import createBaseRule from './createBaseRule'
+
+/**
+ * Creates formatted frames where every frame value is a rule instance.
+ */
+function formatFrames(frames, rule) {
+  const newFrames = Object.create(null)
+  for (const name in frames) {
```

Comment:
This may be the case: https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#52-the-object-being-iterated-is-not-a-simple-enumerable